<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveFileButton">
        <property name="text">
         <string>Save File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="nextDayButton">
      <property name="text">
       <string>Next Day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previousDayButton">
      <property name="text">
       <string>Previous Day</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="durationButton">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="leapYearButton">
      <property name="text">
       <string>Is Leap Year</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="weekNumberButton">
      <property name="text">
       <string>Week Number</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editDateButton">
      <property name="text">
       <string>Edit date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addDateButton">
      <property name="text">
       <string>Add Date (below)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="daySpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="monthSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="yearSpinBox">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="yourBirthdayDateLabel">
      <property name="font">
       <font>
        <family>URW Bookman</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Next/previous birthday date (format dd.mm.yyyy):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="birthdayLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="birthdayButton">
      <property name="text">
       <string>How many days to Birthday? :)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="daysTillBL">
        <property name="font">
         <font>
          <family>URW Bookman</family>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Days till your birthday (from today):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="daysTillBirthdayLabel">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>no information yet</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
